.github/workflows/Publish to Azure Artifacts.yml

```diff
@@ -55,6 +56,56 @@ jobs:
       with:
           repository: Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation
           path: Ed-Fi-ODS-Implementation/
+    - name: Verify dependent workflows are success or not 
+      run: |
+        $workflowNames = @("InitDev, Smoke Tests.yml", "InitDev Postgres, Unit tests, Integration tests.yml", "InitDev, MultiTenancy.yml")
+        $sha = "${{ github.event.workflow_run.head_sha }}"
+        Write-Host "sha is $sha"
+
+          Write-Host "Checking workflow statuses..."
+          $allSuccess = $true
+
+          foreach ($workflow in $workflowNames) {
+            
+            $url = "https://api.github.com/repos/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/actions/workflows/$workflow/runs?head_sha=$sha"
+            $headers = @{
+                    Authorization = "Bearer $env:EDFI_ODS_TOKEN"
+                    Accept = "application/vnd.github.v3+json"
+            }
+            
+            $response = Invoke-RestMethod -Uri $url -Headers $headers
+
+            if ($response.workflow_runs.Count -eq 0) {
+              Write-Host "No runs found for workflow: $workflow"
+              $allSuccess = $false
+              break
+            }
+
+            $latestRun = $response.workflow_runs | Sort-Object -Property created_at -Descending | Select-Object -First 1
+            if ($latestRun.conclusion -ne "success") {
+              Write-Host "latest Run status is $latestRun.status"
+              $allSuccess = $false
+              break
+            }
+          }
+
+          if ($allSuccess) {
+            Write-Host "All workflows completed successfully."
+            break
+          }
+
+        if (-not $allSuccess) {
+          Write-Host "one or more workflows failed.So cancelling the build"
+          exit 1
+        }
+      shell: pwsh
+    # Adding a step to ensure the build job fails if it was skipped
+    - name: Fail the build job if skipped
+      if: ${{ needs.FindStandardAndExtensionVersions.result == 'skipped' }}
+      run: |
+        echo "Build job was skipped due to a failure in the previous job."
+        exit 1
+      shell: pwsh
```

Comment:
These changes aren't needed since [ahmadnassri/action-workflow-run-wait](https://github.com/ahmadnassri/action-workflow-run-wait) that we use below already does these checks internally.

We just need to remove the `if` from my comment above so that the checks gets executed.